adreset2/apiv1/ad_config.py

```diff
@@ -0,0 +1,56 @@
+"""
+Author: StackFocus
+File: configs.py
+Purpose: The configs API for ADReset2 which allows an admin to update PostMaster configurations
+"""
+from flask import request
+from flask_login import login_required, current_user
+from adreset2 import db
+from adreset2.models import AdConfigs
+from adreset2.errors import ValidationError, GenericError
+from adreset2.decorators import json_wrap, paginate
+from adreset2.apiv1 import apiv1
+# This broad import statement is so that functions from utils can be mocked in the Blueprint
+import adreset2.utils
```

Comment:
Can you explain this more to me? 

Is this because if you import the functions directly you can't mock them?